### PR TITLE
Add skeleton code for UI, with general methods

### PR DESCRIPTION
**Pull request by alstontham**
Created the UI class, in charge of handling all of the results that the user will see on the command line.

### CODE REVIEW

File: src/main/java/athena/Ui.java [hunk 1]
```diff
@@ -0,0 +1,2 @@
+package athena;public class Ui {
```
Comment (wish2023) [inline src/main/java/athena/Ui.java]: Make sure to have this on two seperate lines

Comment (alstontham) [reply src/main/java/athena/Ui.java]: Hmm not sure why it shows up like this on github but I definitely had it on two separate lines in my IDE

File: src/main/java/athena/Ui.java [hunk 2]
```diff
@@ -1,2 +1,93 @@
...
+
+    }
+
+    //todo: could also include a logo to make it more friendly
+    public void printWelcomeMessage() {
+        System.out.println("Hello! I'm the God of Wisdom and War, the mighty ATHENA!\n" + "...\n"
+                + "Okay okay I'm not a God but I am your Automated Timetable Helper Encourager n' Assistant!\n"
+                + "What can I do? Are you challenging me to a duel?\n" + "...\n"
+                + "Oh you mean in terms of tasks? Just type \"help\" to witness my mighty repertoire!\n"
+                + "So, what would you like to do today?\n");
```
Comment (daniellimws) [inline src/main/java/athena/Ui.java]: HAHA nice

**Review by daniellimws — APPROVED** [review]
Looks good! Hope we can merge without build errors after merging in the code for `Task` and `TaskList`!

File: src/main/java/athena/Ui.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    /**
+     * Prints the first message that the user sees upon launching the application.
+     */
+    public void printWelcomeMessage() {
+        System.out.println("Hello! I'm the Goddess of Wisdom and War, the mighty ATHENA!\n" + "...\n"
+                + "Okay okay I'm not a Goddess but I am your Automated Timetable Helper Encourager n' Assistant!\n"
+                + "What can I do? Are you challenging me to a duel?\n" + "...\n"
+                + "Oh you mean in terms of tasks? Just type \"help\" to witness my mighty repertoire!\n"
+                + "So, what would you like to do today?\n");
+    }
```
Comment (wish2023) [inline src/main/java/athena/Ui.java:20]: Brilliant

File: src/main/java/athena/Ui.java [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+                + "To edit a task (parameters in square brackets are optional fields):\n"
+                + "edit INDEX [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]\n"
+                + "e.g. edit 2 n/Assignment1 t/1100 d/16/09 D/2 r/Monday i/high a/Refer to lecture notes\n\n"
+                + "To delete a task:\n"
+                + "delete INDEX\n"
+                + "e.g. delete 2\n\n"
+                + "To say farewell to me, ATHENA:\n"
+                + "exit\n"
+                + "But why would you want to leave me anyways? Hmph.\n");
+    }
```
Comment (wish2023) [inline src/main/java/athena/Ui.java:91]: Well organised the code looks really neat!

**Review by wish2023 — APPROVED** [review]
I love the humour omg

**Pull request merged**